pkg/pipeline/extract/aggregate/aggregate.go

```diff
@@ -48,6 +48,8 @@ type Aggregate struct {
 type GroupState struct {
 	normalizedValues NormalizedValues
 	RecentRawValues  []float64
+	recentOpValue    float64
+	recentCount      int
```

Comment:
s/recentCount/rescentOpCount

Logic: To align with `recentOpValue`

Reply:
I don't agree. In a later PR I'll add the `raw_values` operation to support histograms. 
That PR will redefine when the field `RecentRawValues` is set. Currently, it is always set.
That PR will cause it to be set only when the new `raw_values` operation is used.
Moreover, when `raw_values` is used, `recentOpValue` has no meaning.

To conclude, we will have 3 related fields:
1. recentRawValues
2. recentOpValue
3. recentCount

The 1st will be set only for `raw_values` operation,
The 2nd will be set for the other operations,
And the 3rd will be set for all operations.

I think that the common prefix should be `recent` rather than `recentOp`

Reply:
OK :-) I am convinced